src/extension/openFileAtLocation.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {RemoteExtension} from "../common/remoteExtension";
+import {ReactNativeProjectHelper} from "../common/reactNativeProjectHelper";
+import {InternalErrorCode} from "../common/error/internalErrorCode";
+import {ErrorHelper} from "../common/error/errorHelper";
+import * as path from "path";
+
+/* Usage:
+...path\openFileAtLocation.js filename:lineNumber
+...path\openFileAtLocation.js filename
+*/
+
+if (process.argv.length < 3) {
+    throw "Wrong number of parameters provided. Please refer to the usage of this script for proper use.";
+}
+const fileInfo = process.argv[2].split(":");
```

Comment:
I'm inclined to go with something more like

```const fullpath = process.argv[2];
const filename = path.basename(fullpath);
const dirname = path.dirname(fullpath);
const fileInfo = filename.split(":");```

and reconstructing the full name by taking the dirname and the non-line-number part of the filename together again.